### PR TITLE
N41: Tiny Glade

### PR DESCRIPTION
**Pull request by anopara**
Part of #1233

### CODE REVIEW

File: content/news/041/index.md [hunk 1]
```diff
@@ -69,6 +69,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
 ## Game Updates
 
+### Tiny Glade
+
+![tiny_glade](tiny_glade_key_art_small.png)
+
+[@anopara]'s and [@h3r2tic]'s tiny building game
+now finally has a name! It's [Tiny Glade] 🏰🌿🐑
+
+Tiny Glade a small relaxing game about doodling
```
Comment (17cupsofcoffee) [inline content/news/041/index.md:79]:
```suggestion
Tiny Glade is a small relaxing game about doodling
```

File: content/news/041/index.md [hunk 2]
```diff
@@ -69,6 +69,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+castles. Explore gridless building chemistry, and
+watch the game carefully assemble every brick, pebble
+and plank. There's no management, combat, or wrong
+answers - just kick back and turn forgotten meadows
+into lovable dioramas. Wishlist on 🚂[Steam]💨
+
+[@anopara]: https://twitter.com/anastasiaopara
+[@h3r2tic]: https://twitter.com/h3r2tic
+[Tiny Glade]: https://store.steampowered.com/app/2198150/Tiny_Glade/
+[Steam]: https://store.steampowered.com/app/2198150/Tiny_Glade/
```
Comment (17cupsofcoffee) [inline content/news/041/index.md:89]:
```suggestion
into lovable dioramas. Wishlist on 🚂[Steam][Tiny Glade]💨

[@anopara]: https://twitter.com/anastasiaopara
[@h3r2tic]: https://twitter.com/h3r2tic
[Tiny Glade]: https://store.steampowered.com/app/2198150/Tiny_Glade/
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good :+1: I made a few tweaks and took out the emoji (just for style consistency with the other sections).

**Comment by ozkriff** [comment]
> .png
> +1.67 MB

:crying_cat_face:

**Pull request merged**